examples/File.tsx

```diff
@@ -0,0 +1,26 @@
+// import fs from 'fs';
+// import path from 'path';
+import React from 'react';
+import { Button, ButtonGroup, File } from '../src';
+
+// const file = fs.createReadStream(path.join(__dirname, 'Audio.js'));
```

Comment:
???

Reply:
It is needed for example, to not add real file